bundles/thunk_bundle.js

```diff
@@ -6847,6 +6847,17 @@ module.exports = (() => {
     }
   }
   __name(findAndExecutePackFunction, "findAndExecutePackFunction");
+  function getSelectedAuthentication(manifest, authenticationName) {
+    const { defaultAuthentication, adminAuthentications } = manifest;
+    if (!authenticationName || authenticationName === "defaultUserAuthentication" /* Default */) {
```

Comment:
nit: I think we have `ReservedAuthenticationNames.Default` that you can use here

Reply:
Oh yeah this is the autogenerated .js file, ` thunk.ts` is using that constant.